CHANGELOG.md

```diff
@@ -1,3 +1,57 @@
+## 0.17.0 / 2016-01-27
+
+This version no longer works with Alertmanager 0.4.0 and earlier!
+The alerting rule syntax has changed as well but the old syntax is supported
+up until version 0.18.
+
+* [CHANGE] Integrate with Alertmanager 0.1.0 and higher
+* [CHANGE] Degraded storage mode renamed to rushed mode
+* [CHANGE] New alerting rule syntax
+* [CHANGE] Add label validation on ingestion
+* [CHANGE] Regular expression matchers in PromQL are anchored
```

Comment:
This required changes to tens of rules and alert expressions and broke an unknown number of dashboard expressions. I'd recommend to explain the implications with an example in more detail above.